central/scannerdefinitions/handler/handler.go

```diff
@@ -341,34 +330,39 @@ func (h *httpHandler) openOfflineDefinitions(ctx context.Context, t updaterType,
 		// We want to find a specific file inside the toplevel online file.
 		fallthrough
 	case mappingUpdaterType:
-		// search mapping file
+		// Search mapping file.
 		fileName := filepath.Base(opts.fileName)
 		targetFile, _, err := h.openFromArchive(offlineBlob.Name(), fileName)
 		if err != nil {
 			return nil, err
 		}
 		offlineFile = &vulDefFile{File: targetFile, modTime: offlineBlob.modTime}
 	case vulnerabilityUpdaterType:
-		// check version information in manifest
+		// Check version information in manifest.
 		mf, _, err := h.openFromArchive(offlineBlob.Name(), scannerV4ManifestFile)
 		if err != nil {
+			log.Errorf("open manifest: %v", err)
```

Comment:
Yeah, I've found that when opening files within the archive, this will generate a "file not found" kinda error, and that will be captured up the call chain and returned as "definitions not found"; Although I am OK returning 404 for this particular scenario, I wanted to leave something in the logs to trace it, since missing files in the offline bundle archive is odd.